stories/utils/generate-widget-stories.js

```diff
@@ -210,24 +212,27 @@ export function generateReportBasedWidgetStories( {
 		...customVariants,
 	};
 
-	let widget;
+	const slugs = moduleSlugs.map( ( slug ) => {
+		return `${ slug }-widget`;
+	} );
+	const slug = slugs.join( ' ' );
+
+	const widgetComponentProps = getWidgetComponentProps( slug );
+
+	let widgetElement = <Component { ...widgetComponentProps } />;
```

Comment:
Let's leave it in the `else` statement where it was before:

```javascript
let widgetElement;

if ( wrapWidget ) {
    widgetElement = ( ... );
} else {
    const slug = moduleSlugs.map( ( slug ) => `${ slug }-widget` ).join( ' ' );
    const widgetComponentProps = getWidgetComponentProps( slug );
    widgetElement = <Component { ...widgetComponentProps } />;
}
```

Reply:
Switched back in.